ember-cli-build.js

```diff
@@ -5,7 +5,7 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   let app = new EmberAddon(defaults, {
     'ember-cli-babel': {
-      throwUnlessParallelizable: true,
+      throwUnlessParallelizable: false,
```

Comment:
You'll want to straighten this out before considering a merge. The benefits of TS shouldn't have to come at the cost of serializing previously parallel parts of the build.